website/docs/en/api/javascript-api/types.mdx

```diff
@@ -122,6 +122,10 @@ The type of build target.
 import type { RsbuildTarget } from '@rsbuild/core';
 ```
 
+import EnvironmentContext from '@en/shared/environmentContext.mdx';
```

Comment:
Can we create a standalone page for `environmentContext` and link to it.

It is not recommended to use MDX fragment as it can not generate the right sidebar as expected.

Reply:
everything looks fine 😂 
![20240703-172035](https://github.com/web-infra-dev/rsbuild/assets/22373761/1d681e9f-c481-4462-bce1-280cf2a795a0)

Reply:
okay, maybe fixed

Reply:
`environmentContext` is an important plugin API, I don't think it should be placed at the types page.